bower.json

```diff
@@ -6,13 +6,17 @@
   "dependencies": {
     "angular": "~1.5",
     "angular-animate": "~1.5",
+    "angular-file-upload": "1.1.5",
```

Comment:
Do we need this right now?  Do you have the profile image upload working?  I would rather switch to ng-file-upload as it has more of a following